packages/frontend/src/components/TableConfigPanel/GeneralSettings.tsx

```diff
@@ -151,6 +153,18 @@ const GeneralSettings: FC = () => {
                         setHideRowNumbers(!hideRowNumbers);
                     }}
                 />
+
+                {localStorage.getItem('feat/metrics-as-rows') === 'true' &&
```

Comment:
feature flag ? is this ok to merge it ?

Reply:
I think so, we wanted to ship this in increments. If you prefer we could build off a feature branch but this is how we've done it before. What do you think would be better? I prefer shipping it in small amounts, which reduces the chance we merge a huge PR with a larger surface for errors

Reply:
I think this is a perfect usecase for the localstorage feature flags :ok_hand: